src/helpers/snapshot.ts

```diff
@@ -4,7 +4,10 @@ import { apolloClient } from '@/helpers/apollo';
 import { PROPOSAL_QUERY, VOTES_QUERY } from '@/helpers/queries';
 import cloneDeep from 'lodash/cloneDeep';
 
-export async function getProposalVotes(proposalId: string, first = 20000) {
+export async function getProposalVotes(
+  proposalId: string,
+  { first = 20000, voter = '', skip = 0 }
```

Comment:
```suggestion
  { first, voter, skip }: any = { first: 20000, voter: '', skip: 0 }
```

This way you can omit the empty `{}` in SpaceProposal.vue#130.